From 7f9eafd04711e8a3f666a02d2fa50b931f578832 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Fri, 9 Feb 2018 15:30:39 +0800
Subject: [PATCH] ARM: rockchip_defconfig: remove NETFILTER_XT_MATCH_QUOTA2_LOG

bc86c1de1ded ("ANDROID: android-base.cfg: remove NETFILTER_XT_MATCH_QUOTA2_LOG")

There are currently a couple different implementations for this
functionality. Until things are unified, remove the requirement
for this kernel config.

Change-Id: Ia3f515452871118dab4b8688ff9fd16e87beb9b6
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm/configs/rockchip_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index ea2042768b90..226eef102434 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -150,7 +150,6 @@ CONFIG_NETFILTER_XT_MATCH_PKTTYPE=y
 CONFIG_NETFILTER_XT_MATCH_QTAGUID=y
 CONFIG_NETFILTER_XT_MATCH_QUOTA=y
 CONFIG_NETFILTER_XT_MATCH_QUOTA2=y
-CONFIG_NETFILTER_XT_MATCH_QUOTA2_LOG=y
 CONFIG_NETFILTER_XT_MATCH_SOCKET=y
 CONFIG_NETFILTER_XT_MATCH_STATE=y
 CONFIG_NETFILTER_XT_MATCH_STATISTIC=y
-- 
2.35.3

